content/engine/swarm/networking.md

```diff
@@ -239,6 +239,13 @@ network create`. This enables IPSEC encryption at the level of the vxlan. This
 encryption imposes a non-negligible performance penalty, so you should test this
 option before using it in production.
 
+> **Warning**
+>
+> The automatically created [ingress network must be customized](#customize-ingress)
+> to enable encryption. By default, all ingress traffic is unencrypted, as encryption
+> is a network-level option.
+{ .warning }
+
```

Comment:
Maybe this can be a note, not a warning

Reply:
I have no strong opinion here, other than wanting to avoid surprises for users. Happy to do whatever the docs team thinks is best.

Reply:
I also think a note is fine for this; let me amend